### DISCUSSION TITLE
Github 直接搜索不显示工具条

### DISCUSSION THREAD

**Discussion by benzBrake** [hoothin/SearchJumper#84] [Q&A]
RT。但是从别的搜搜引擎跳转github后可以显示工具条，请问怎么解决

**Comment by hoothin** [comment]
如果已经匹配上当前站点就不会再次匹配了。不知道你用的是什么样的引擎设置，应该是多个指向github的引擎起冲突了

**Comment by benzBrake** [comment]
> 用的是什么样的引擎设置
默认配置
直接搜索的链接是 https://github.com/search?q=yt-dlp&type=repositories
从别的搜索引擎跳转过来的跳转的连接是 https://github.com/search?utf8=%E2%9C%93&q=yt-dlp&type=repositories

**Comment by hoothin** [comment]
这样啊，你可以导入[我的引擎](https://github.com/hoothin/SearchJumper/discussions/65)。
或者打开配置页，点开开发分组下的github，添加`根据 URL 正则匹配当前引擎`-`https://github\.com/search.*[&\?]q=`
